backend/corpora/common/utils/aws.py

```diff
@@ -89,3 +89,14 @@ def _restore(self):
     def _debug(self, message):
         if self.debug_logging:
             print(message)
+
+
+def delete_many_from_s3(bucket_name: str, dataset_uuid: str) -> None:
+    """
+    This deletes everything with a specific prefix from the given bucket
+
+    """
+    assert dataset_uuid
```

Comment:
Don't use assert in production code. Use and `if not dataset_uuid: raise ValueError` in this case.  Using asserts in production code can make debug difficult since python removes asserts when converting to when it is compiled. See https://docs.python.org/3/using/cmdline.html#cmdoption-o. We don't want to accidentally delete the contents of a  bucket because someone used the `-o` flag before deploying.